### DISCUSSION TITLE
Song plays vs album plays

### DISCUSSION THREAD

**Discussion by tehabe** [navidrome/navidrome#3526] [Q&A]
I'm a bit confused about the number of plays. I have an album with a lot of plays, but the songs have a very much lower number. When is a album being counted as played when is just the song counted?

**Comment by metalheim** [comment]
See https://www.navidrome.org/docs/usage/configuration-options/#advanced-configuration

By default, the Album Playcount is the SUM of all song playcounts in it.
You can change navidrome configuration value `AlbumPlayCountMode` to `normalized` and navidrome will divide the Album Playcount by the number of songs.

Some more context here: https://github.com/navidrome/navidrome/pull/2803
https://github.com/navidrome/navidrome/discussions/2711
https://github.com/navidrome/navidrome/issues/1032
